example/expression/access_c.py

```diff
@@ -0,0 +1,143 @@
+import sys
```

Comment:
This is a really nice example, but it is hard to see the purpose, then why specific reduction are used and finally how the whole thing is use.
As this will be the referring example of use of ObjC features, it will probably be more accessible with a few more comment, and a summary / warm-up at the beginning.
In addition, an example of using `c_to_expr` API could also be great, and take away the sacred aura from using this feature.